diff --git a/node_modules/react-native/Libraries/Image/RCTImageLoader.mm b/node_modules/react-native/Libraries/Image/RCTImageLoader.mm
index 3571647..de8ff9a 100644
--- a/node_modules/react-native/Libraries/Image/RCTImageLoader.mm
+++ b/node_modules/react-native/Libraries/Image/RCTImageLoader.mm
@@ -1040,12 +1040,12 @@ - (BOOL)canHandleRequest:(NSURLRequest *)request

 - (id)sendRequest:(NSURLRequest *)request withDelegate:(id<RCTURLRequestDelegate>)delegate
 {
-  __block RCTImageLoaderCancellationBlock requestToken;
+  __block RCTImageLoaderCancellationBlock requestToken = ^{};
   requestToken = [self loadImageWithURLRequest:request callback:^(NSError *error, UIImage *image) {
     if (error) {
       [delegate URLRequest:requestToken didCompleteWithError:error];
       return;
-    }
+    }

     NSString *mimeType = nil;
     NSData *imageData = nil;
diff --git a/node_modules/react-native/scripts/.packager.env b/node_modules/react-native/scripts/.packager.env
new file mode 100644
index 0000000..361f5fb
--- /dev/null
+++ b/node_modules/react-native/scripts/.packager.env
@@ -0,0 +1 @@
+export RCT_METRO_PORT=8081
